docs/prerequisites.md

```diff
@@ -1,93 +1,86 @@
 # Pre-Requisites
 
-- [AWS credentials](#aws-credentials)
-    - [For non-STS clusters](#for-non-sts-clusters)
-    - [For STS clusters](#for-sts-clusters)
-        - [Option 1. Using ccoctl](#option-1-using-ccoctl)
-        - [Option 2. Using the AWS CLI](#option-2-using-the-aws-cli)
+- [IAM Role for STS clusters](#iam-role-for-sts-cluster)
+    - [Option 1. Using ccoctl](#option-1-using-ccoctl)
+    - [Option 2. Using the AWS CLI](#option-2-using-the-aws-cli)
 - [VPC and Subnets](#vpc-and-subnets)
     - [VPC](#vpc)
     - [Subnets](#subnets)
         - [Public subnets](#public-subnets)
         - [Private subnets](#private-subnets)
 
-## AWS credentials
-Additional AWS credentials are needed for the operator to be successfully installed. This is needed to interact with subnets and VPCs.
+# IAM Role for STS clusters
+Additional IAM Role is needed for the operator to be successfully installed. This is needed to interact with subnets and VPCs.
+The operator will generate `CredentialsRequest` with this role to self bootstrap with AWS credentials.
 
-### For non-STS clusters
-
-1. Create AWS credentials profile for the operator:
-
-    ```bash
-    cat << EOF > credentials
-    [default]
-    aws_access_key_id=${AWS_ACCESS_KEY_ID}
-    aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
-    EOF
-    
-    oc create secret generic aws-load-balancer-operator \
-    -n aws-load-balancer-operator \
-    --from-file=credentials=credentials
-    ```
-  
-2. Alternatively use the `CredentialsRequest` resource in the `hack` directory:
-
-   ```bash
-   oc apply -f https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
-   ```
-
-### For STS clusters
-
-There are two options for creating the credentials secret:
+There are two options for creating the IAM role:
 - Using a pre-defined `CredentialsRequest`.
 - Using pre-defined AWS manifests.
 
 For handling `CredentialsRequests`, the cloud credential operator utility, [ccoctl](https://docs.openshift.com/container-platform/latest/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts), can be utilized.
 If you prefer not to use `ccoctl`, or your system doesn't support it, the AWS CLI can be an alternative.
 
-#### Option 1. Using `ccoctl`
+## Option 1. Using `ccoctl`
 
 1. [Extract and prepare the `ccoctl` binary](https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
```

Comment:
I agreed with you when we put `latest` in a link to the ALBO doc in the OperatorHub's installation page. Because it was our doc and `main` branch should always be consistent with the latest docs.
However here we have a doc which is out of our control, so it may change to something which would not work anymore. So I'd prefer to fix the link to a concrete version.